newton/_src/sim/builder.py

```diff
@@ -5453,32 +5454,34 @@ def finalize(self, device: Devicelike | None = None, requires_grad: bool = False
             m.shape_collision_group = wp.array(self.shape_collision_group, dtype=wp.int32)
 
             # ---------------------
-            # Compute SDFs for mesh shapes (opt-in feature via enable_mesh_sdf_collision)
+            # Compute SDFs for mesh shapes (per-shape opt-in via sdf_max_dims)
             from ..geometry.sdf_utils import SDFData, compute_sdf, create_empty_sdf_data  # noqa: PLC0415
             from ..geometry.types import GeoType  # noqa: PLC0415
```

Comment:
While you're at it, this include shouldn't be necessary. We import GeoType at the top of this file already.